tracer/src/Datadog.Trace/Debugger/Expressions/ProbeExpressionParser.Dump.cs

```diff
@@ -26,14 +26,14 @@ private Expression DumpExpression(Expression expression, List<ParameterExpressio
             return Expression.Call(expression, GetMethodByReflection(typeof(object), nameof(object.ToString), Type.EmptyTypes));
         }
 
-        var stringConcat = GetMethodByReflection(typeof(string), nameof(string.Concat), new[] { typeof(object[]) });
+        var ifNull = Expression.Equal(expression, Expression.Constant(null));
 
         if (IsSafeException(expression.Type))
         {
             // for known Exception types we can assume it's safe to call .Message
             // whereas the others might have overriden it in a way that could cause side effects
+            var stringConcat = GetMethodByReflection(typeof(string), nameof(string.Concat), new[] { typeof(object[]) });
```

Comment:
Nit: Not sure it's relevant for this PR but - do we want to cache aside all the `MethodInfo` types we use throughout the processing of expression?
I see many calls to `GetMethodByReflection`, many of which could be easily kept aside since they won't change, as in this case.

Reply:
You are right about it and they are already have cached :)